src/server.rs

```diff
@@ -605,6 +643,11 @@ impl RollupBoostServer {
 
         let (l2_payload, builder_payload) = tokio::join!(l2_client_future, builder_client_future);
         let (payload, context) = match (builder_payload, l2_payload) {
+            (Ok(Some(builder)), Ok(l2)) => {
```

Comment:
have you tested that the metrics show up? wondering if you need to add these metrics to the instrument macro
```
    #[instrument(
        skip_all,
        err,
        fields(
            otel.kind = ?SpanKind::Client,
            target = self.payload_source.to_string(),
            head_block_hash = %fork_choice_state.head_block_hash,
            url = %self.auth_rpc,
            code,
            payload_id
        )
    )]
```

Reply:
Great question, I think I missed this because I did not test. What is your workflow for testing metrics like this in a local dev environment?

Reply:
we spin up a local op-stack with [builder-playground](https://github.com/flashbots/builder-playground) and then curl the metrics endpoint

Reply:
For posterity, the setup is similar to this: 
https://github.com/flashbots/rbuilder/blob/develop/crates/op-rbuilder/README.md#local-devnet